src/test/resources/org/rust/ide/actions/fixtures/go_to_super_mod/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod bar;
```

Comment:
I think you can drop this file and just add `mod bar;` to `main.rs`. This way, there should be no need to make `trimDir` protected